db.ts

```diff
@@ -72,4 +72,20 @@ const createNewHighScore = async (user: User, typeOfQuiz: string, newHighScore:
     }
 }
 
-export { connect, createUser, getUser, createNewHighScore }
\ No newline at end of file
+const deleteBlacklist = async (user: User, quoteId: string) => {
+    try {
+        await client.db("TheOneQuiz").collection("Users").updateOne({ username: user.username }, { $pull: { blacklist: { quoteId: quoteId } } });
```

Comment:
Here as well, don't use "quoteId" as a parameter, but "quote_id", otherwise it does not work

```suggestion
        await client.db("TheOneQuiz").collection("Users").updateOne({ username: user.username }, { $pull: { blacklist: { quote_id: quoteId } } });
```